### PR TITLE
Newsweek Landing page

### PR DESCRIPTION
**Pull request by Haroonabdulrazaq**
This project is focused on building responsiveness(i.e across all media sizes) using Bootstrap, by cloning Newsweek landing page.

PLEASE check this former REPO for more detail on this project:
https://github.com/Haroonabdulrazaq/Newsweek-clone/pull/5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
-# Newsweek
-This project is focused on building responsiveness(i.e across all media size) using Bootstrap, by cloning Newsweek landing page
+# Newsweek-clone
+Bootstrap Project [Microverse]
+# Newsweek Clone
```
Comment (elmejdki) [inline README.md]: you have two titles please remove one.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1361 @@
...
+
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;700&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>Newsweek - News, Analysis, Politics, Business, Technology</title>
+</head>
+
+<body>
+  <!-- ++++++++++++++++++++++++++++++++++++++++++++++++++
+    Navigation 
+    ++++++++++++++++++++++++++++++++++++++++++++++++++ -->
```
Comment (elmejdki) [inline index.html:20]: Please remove all comments from your HTML and CSS code:
image

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
+  line-height: 1em;
+}
+
+/* Navigation Styles */
```
Comment (elmejdki) [inline css/style.css:20]: Please remove all the comments for your CSS and HTML files:
image

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Here I'm back again, now we will see what we have here, thank you for making the previous requirements :tophat: :blush:.

- Please put these two links "login and subscribe" into the right extreme corner.
image

- Please fix the top menu, on small screen devices it breaks, I think that because you are using a big search input try to make it small and see what will happen:
image

- fix menu on tablet view it should be a little bit ticker, and also add some space between it and the articles:
#### your version:
image
#### original page:
image


- You have some overlapping images over text in more stories section please fix it:
#### Desktop view:
image
#### Tablet View same for Mobile view:
image

- there is a huge space between In The Magazine section and Editor's Pick please fix it, I noticed that you are using vh in the height of the in-the-magazine class change it into a fixed height value:
image

- fix this overlapping on subscribe section:
image

- I see that you are not using semantic HTML please add it to your project, you should use tags like header, nav, section, main, article, ...etc. [click here](https://www.w3schools.com/html/html5_semantic_elements.asp) for more details.

- Please validate your HTML with [HTML validator](https://validator.w3.org/) and fix all the errors, here is some of what I found there:
image

### please fix all the required changes before you request another code review.
_if you have any questions don't hesitate to contact me on slack_

File: css/style.css (hunk 3 above)
Comment (Haroonabdulrazaq) [reply css/style.css:20]: The inline comment here is talking about commenting codes like(//console.log("Hello")) and not segmentations like <--Header navigation-->

**Comment by santiago-dev96** [comment]
# PR under review by @santiago-rodrig (TSE)

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🔧

Good job so far, but there are some things to fix.

1. The distance between your navigation bar and the main content is not enough (small and medium screens).

Screen Shot 2020-06-12 at 8 35 20 AM

2. The subscribe section is not following the design of the original page (small screens).

**Original**

Screen Shot 2020-06-12 at 8 27 51 AM

**Yours**

Screen Shot 2020-06-12 at 8 27 59 AM

3. The footer social media links are not following the design of the original page (small screens).

**Original**

Screen Shot 2020-06-12 at 8 29 08 AM

**Yours**

Screen Shot 2020-06-12 at 8 29 16 AM

## What to do now?

Implement the required changes and submit another CRR (Code Review Request).

## How can you reach out to me?

I'm available in slack as **srodrig**, happy coding! :wave:

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks: I love how clean and modular your code is. Please keep it up.
##
### The [clone](https://raw.githack.com/Haroonabdulrazaq/Newsweek/homepage/index.html) shows good understanding and usage of bootstrap. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
-  The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @Haroonabdulrazaq! :+1:

## OPTIONAL CHANGES 
- [ ] We can improve on the centring of the page using the `container`  class to centre all the pages's content to appear strikingly similar to the source page. Please look at the difference below. 👇 

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://raw.githack.com/Haroonabdulrazaq/Newsweek/homepage/index.html)|
|:---:|:---:|
|source|clone|

### Note:  Kindly consider this change to be strictly optional, thank you!
- You don't need to submit for another code review. Consider this complete and you can now merge to master. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**